README.md

```diff
@@ -17,15 +17,30 @@
 [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r.svg?style=svg)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-r) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-r/depshield.svg)](https://depshield.github.io)
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
+> **What's New** R is now part of Nexus Repository Manager. Version 3.20+ includes the R plugin by default. At the same time it is still open for community. 
```

Comment:
Minor: I'd like to make it clear that this is community project on GitHub is the "source of truth" when it comes to source code. I want to avoid someone wondering if they need to dig through the 'nexus-public' code, or look here. Could you come up with some more explicit comment than "still open for the community"? 

Maybe something like: "The source code for the R plugin continues to live in this community project."

Reply:
will be updated in the release note section. Thanks! @bhamail